lib/commands/test-init.ts

```diff
@@ -37,9 +37,7 @@ class TestInitCommand implements ICommand {
 		const dependencies = this.frameworkDependencies[frameworkToInstall] || [];
 		const modulesToInstall: IDependencyInformation[] = [
 			{
-				name: 'karma',
-				// Hardcode the version unitl https://github.com/karma-runner/karma/issues/3052 is fixed
-				version: "2.0.2"
+				name: 'karma'
```

Comment:
Maybe just update the version. It will be bad if a new karma version breaks our functionality.

Reply:
https://github.com/NativeScript/nativescript-cli/issues/4390